scripts/dns_rr_configs.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+
+#####################################
+# Script to configure dns round robin
+#####################################
+
+import argparse
+import sys
+import os
+from typing import List
+
+def chk_ip(ip: str) -> bool:
+    octs = ip.split(".")
+    if len(octs) != 4:
+        return False
+    for o in octs:
+        try:
+            ioct = int(o)
+            if ioct > 255 or ioct < 0:
+                return False
+        except ValueError:
+            return False
+    return True
+
+def chk_dir(d: str) -> bool:
+    return os.path.isdir(d) and os.access(d, os.W_OK)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        prog=sys.argv[0],
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description="""Configure BIND dns server to round robin IPs.""",
+        epilog=f"""Examples:
+To configure with default params: zone seagate.com and services s3, iam
+        #> {sys.argv[0]} --ip 192.168.1.0 192.168.1.1
+
+To configure for different zones and services
+        #> {sys.argv[0]} --ip 192.168.1.0 192.168.1.1 --zone test.zone --services srv1 srv2
+        """)
+
+    argz = parser.add_argument("-z", "--zone", type=str, required=False, default="seagate.com",
```

Comment:
should we not use "seagate" for the default zone name ?

I am thinking if this script runs in customer environment (correct me if I am wrong), then their zone should not be set to "seagate.com" (even if its harmless), but to some more generic name, like myzone.com.

Reply:
In this case I propose we don't have any default at all and make this option a mandatory argument?  @amitwac2608 @dsurnin

Reply:
this script configures local dns server and configures machine it was run on to use this server.
script does not provide any production ready server configuration just a fast and simple test environment.
it should not be recommended for end-user usage.

Reply:
Ok, we can go ahead then and merge this.